src/components/Frame/types.ts

```diff
@@ -9,9 +9,7 @@ export interface FrameManager {
   stopLoading(): void;
 }
 
-export interface FrameContext {
-  frame: FrameManager;
-}
+export interface FrameContext extends FrameManager {}
```

Comment:
Should `FrameManager` just be moved here instead of extending?

Reply:
👍 Doesn't seem like it's being used anywhere else